test/test_unary_ufuncs.py

```diff
@@ -212,8 +212,9 @@ def _fn(t):
         for alt, inplace in ((op.get_method(), False), (op.get_inplace(), True),
                              (torch.jit.script(_fn), False)):
             if alt is None:
-                with self.assertRaises(RuntimeError):
+                with self.assertRaisesRegex(TypeError, "'NoneType' object is not callable"):
                     alt(t.clone())
+                continue
```

Comment:
I think we should just continue if alt is `None`.

Here in this case, `angle` doesn't have an inplace variant and hence `alt` is `None` for that case.

Reply:
I think you're right.